tests/Integration/Behaviour/Features/Scenario/Product/update_seo.feature

```diff
@@ -383,3 +384,29 @@ Feature: Update product SEO options from Back Office (BO)
       | redirect_target | clothes                               |
       | redirect_name   | Home > Clothes                        |
       | redirect_image  | http://myshop.com/img/c/{clothes}.jpg |
+
+  Scenario: Empty friendly-urls should be auto-filled using product name value
+    And I add product "product4" with following information:
+      | type        | standard |
+      | name[en-US] |          |
+      | name[fr-FR] |          |
+    Then product "product4" localized "name" should be:
+      | locale | value |
+      | en-US  |       |
+      | fr-FR  |       |
+    When I update product "product4" basic information with following values:
+      | name[en-US] | en product 04 |
+      | name[fr-FR] | fr product 04 |
+    Then product "product4" localized "name" should be:
+      | locale | value         |
+      | en-US  | en product 04 |
+      | fr-FR  | fr product 04 |
+    When I update product product4 SEO information with following values:
+      | link_rewrite[en-US] |  |
+      | link_rewrite[fr-FR] |  |
+    # UpdateProductBasicInformationCommand also modifies link_rewrite, so it was already changed during name update
+    # therefore this test is not the most accurate
+    And product "product4" localized "link_rewrite" should be:
```

Comment:
At least this test validates that you can't empty the link_rewrite when the name is filled